script-library/docker-in-docker-debian.sh

```diff
@@ -159,7 +159,7 @@ fi
 ## Dind wrapper over.
 
 # Start docker/moby engine
-( sudoIf dockerd > /tmp/dockerd.log 2>&1 ) &
+( sudoIf dockerd --dns 168.63.129.16 > /tmp/dockerd.log 2>&1 ) &
```

Comment:
Can we detect if the container is running in Azure and only set this in that case? We want to be sure to allow this image to work well offline / outside of Codespaces as well. But when running in the Codespaces infrastructure, we want it to default to this DNS server.